src/api/lib/middlewares/tokenChecker.js

```diff
@@ -17,6 +17,11 @@ export default async (ctx, next) => {
 
     try {
         ctx.user = await jwt.verify(token, config.apps.api.security.jwt.privateKey);
+
+        if (!ctx.user.expires || new Date(ctx.user.expires) < new Date()) {
```

Comment:
You don't have to check expiration yourself, it is `verify` job (https://github.com/auth0/node-jsonwebtoken#tokenexpirederror)